packages/apps/crm/src/utils/list.js

```diff
@@ -16,10 +16,22 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-export const getLastEvent = listEvent => {
+export const getLastItem = listItem => {
+  if (
+    listItem == null ||
+    Array.isArray(listItem) === false ||
+    listItem.length === 0
+  ) {
+    return null;
+  }
+
+  if (listItem.length === 1) {
+    return listItem[0];
+  }
```

Comment:
If event is in the future, the only item is not last item, it should go to standard filter